### PR TITLE
Use new DESKTOP_APP_USE_PACKAGED_LAZY to reduce maintenance complexity

### LINKED ISSUE #80: Input method fctix5 support
I use fcitx5

But unable to use input method on `org.telegram.desktop`

Maybe need `fcitx5-qt`?

### PR DESCRIPTION
**Pull request by ilya-fedin**
Closes #80

### CODE REVIEW

File: org.telegram.desktop.json [hunk 1]
```diff
@@ -187,12 +81,8 @@
                 {
                     "type": "git",
                     "url": "https://github.com/telegramdesktop/tdesktop.git",
-                    "tag": "v2.1.1",
-                    "commit": "07c8aae225e2f04600a7d5411047d30f4ca5f51c"
-                },
-                {
-                    "type": "patch",
-                    "path": "patches/Fix-decoration-applying.patch"
```
Comment (gasinvein) [inline org.telegram.desktop.json:195]: Remove the patch file, too, if it's not needed.

Comment (ilya-fedin) [reply org.telegram.desktop.json:195]: Oh, looks like my vscode doesn't delete files again

Thanks

**Comment by ilya-fedin** [comment]
After discussing with @gasinvein we split the update and this flag

**Comment by ilya-fedin** [comment]
Since 2.1.5 introduces fcitx5 plugins, this closes #80

**Pull request merged**